src/Tools/dotnet-counters/Program.cs

```diff
@@ -135,6 +177,22 @@ internal class Program
                 Argument = new Argument<bool>(name: "resumeRuntime", getDefaultValue: () => true)
             };
 
+        private static Option MaxHistogramOption() =>
+            new Option(
+                alias: "--maxHistograms",
+                description: "The maximum number of histograms that can be tracked. Tracking more histograms uses more memory in the target process.")
+            {
+                Argument = new Argument<int>(name: "maxHistograms", getDefaultValue: () => 10)
+            };
```

Comment:
good to know about the uint option but I agree given the inferior help text I'll leave my custom validation in place for now.

Reply:
I don't see an overload of Option<int>() that takes a getDefaultValue argument. Perhaps you are thinking of a different version of the System.CommandLine than the one being used by dotnet-counters right now?

Reply:
That's odd. Not a big deal, I compiled this locally and I knew that Arguments were not public in newer releases so I was just suggesting the future working one.

Reply:
That is quite the mystery that it would compile for you and give me an error. Its always possible I messed something up. I'm not going to prioritize looking into it, but as long as it compiles I think it was a beneficial change we could take at any time.